client/src/components/deals.jsx

```diff
@@ -0,0 +1,38 @@
+const React = require('react');
+// const styled = require('styled-components');
+// const Deal = require('./deal.jsx');
+
+const Deals = (props) => (
```

Comment:
If you'd like, you can grab deals directly out of the props with de-structuring.  It would look like the following:
```javascript
const Deals = ({ deals }) => (
```

Then deals can be accessed with `deals` instead of `props.deals`

Reply:
Thank you so much for that tip! Now that I am comfortable looking at de-structuring adding this won't be a big deal. I'll change it for the next PR.